### ISSUE TITLE
@microsoft/fast-react-wrapper is missing "type": "module" in 0.x

### ISSUE THREAD

**Issue by jedwards1211** [microsoft/fast#6528]
# 🐛 Bug Report

When using `"moduleResolution": "NodeNext"`, I get errors inside `@microsoft/fast-react-wrapper@0.1.48` (used by `@vscode/webview-ui-toolkit`) due to TS considering its code to be CommonJS by default.

I'm not sure how to make a PR since the master branch has that package on an alpha version.

**Comment by EisenbergEffect** [comment]
I don't think we can update our previous version libraries to use `"type": "module"`. That was a major change we made for the new versions. I think that would break people. If I recall there were additional reasons why we couldn't do this prior to the new versions. @nicholasrice may recall the details.

**Comment by jedwards1211** [comment]
Huh, seems odd considering that the published source code has `import`/`export` statements anyway?

**Comment by janechu** [comment]
Unfortunately `@microsoft/fast-react-wrapper` is being deprecated, refer to #6955. There are other solutions for wrapping web components for React and it appears React is working on it's own solution, please refer to their documentation for details.
